routing/index_graph_starter.hpp

```diff
@@ -99,6 +94,8 @@ class IndexGraphStarter final
     return segment.GetFeatureId() == kFakeFeatureId;
   }
 
+  Segment const & ConvertSegment(Segment const & segment);
```

Comment:
Comment to the method, please. Please, describe what the method does and when it should be used.